src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -549,8 +550,6 @@ PyMODINIT_FUNC PyInit_pythonmonkey(void)
 
   JS::RealmCreationOptions creationOptions = JS::RealmCreationOptions();
   JS::RealmBehaviors behaviours = JS::RealmBehaviors();
-  creationOptions.setWeakRefsEnabled(JS::WeakRefSpecifier::EnabledWithoutCleanupSome); // enable FinalizationRegistry
-  creationOptions.setIteratorHelpersEnabled(true);
```

Comment:
@caleb-distributive please double-check with change above: why is this new way better?

Reply:
https://github.com/Distributive-Network/PythonMonkey/pull/367#issuecomment-2221218172